.github/workflows/build-test-deploy.yml

```diff
@@ -141,6 +141,7 @@ jobs:
         uses: codecov/codecov-action@v4
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+          flags: unittests
```

Comment:
Should we keep env.vars with uppercase?

Reply:
```suggestion
          FLAGS: unittests
```

Reply:
ah my bad, this is not an env var at all. will fix.

Reply:
Fixed it now